application/account-management/WebApp/src/lib/appInsights/config.tsx

```diff
@@ -0,0 +1,25 @@
+import { ReactPlugin } from "@microsoft/applicationinsights-react-js";
+import { ApplicationInsights } from "@microsoft/applicationinsights-web";
+
+export const reactPlugin = new ReactPlugin();
+
+const appInsights = new ApplicationInsights({
+  config: {
+    appId: "account-management/webapp",
+    instrumentationKey: "webapp",
+    disableInstrumentationKeyValidation: true,
+    endpointUrl: "/api/track",
+    enableAutoRouteTracking: true,
+    autoExceptionInstrumented: true,
+    autoUnhandledPromiseInstrumented: true,
+    autoTrackPageVisitTime: false,
+    isBeaconApiDisabled: true,
+    extensions: [reactPlugin],
+    disableFetchTracking: true,
+    disableAjaxTracking: true,
+  },
```

Comment:
Should we document what these does?

Reply:
the code documentation is pretty good / clear - don't think it's needed?